erna/datacheck_conditions.py

```diff
@@ -1,7 +1,20 @@
 conditions = dict()
 
+def create_condition_set(conditionset=['standard']):
+    """
+    given a list of conditions create a condition set
+    
+    If a given condition start with '@NAME', process NAME as a set of conditions defined in the variabel conditions.
+    """
+    data_conditions = ['fRunTypeName = "data"']
+    for condition in conditionset:
+        if condition.startswith('@'):
+            data_conditions = data_conditions+conditions[condition[1:]]
+        else:
+            data_conditions.append(condition)
+    return data_conditions
+    
 conditions['standard'] = [
-    'fRunTypeName = "data"',
     'fZenithDistanceMean < 30',
```

Comment:
I think this zenith should go away

Reply:
I left it in so it would be backward compatible (I don't want to suprise anyone). So if that is not a concern I will remove it.

Reply:
I would remove it